packages/relay/tests/lib/eth.spec.ts

```diff
@@ -3809,6 +3827,30 @@ describe('Eth calls using MirrorNode', async function () {
       }
       expect(hasError).to.be.true;
     });
+
+    it('should return a computed hash if unable to retrieve EthereumHash from record due to contract revert', async function () {
+      restMock.onGet(accountEndpoint).reply(200, { account: accountAddress });
+    
+      const transaction = {
+        chainId: 0x12a,
+        to: accountAddress1,
+        from: accountAddress,
+        value,
+        gasPrice,
+        gasLimit: maxGasLimitHex,
+      };
+    
+      sdkClientStub.getAccountBalanceInTinyBar.returns(ethers.BigNumber.from('1000000000000000000000'));    
+      const signed = await signTransaction(transaction);
+      const id = uuid();
+    
+      restMock.onGet('network/fees').reply(200, defaultNetworkFees);
+      restMock.onGet('transactions/0.0.902-1684375868-230217103').reply(200, null);
```

Comment:
Based on your fix it seems the actual issue is a null transaction. In that case this call would actually be returning a 404 with a contract revert message

Reply:
Should be addressed in the updates and three new tests.